.azure/default-build.yml

```diff
@@ -22,4 +24,26 @@ jobs:
   
   - ${{ parameters.beforeBuild }}
   
+  - task: CMake@1
+    inputs:
+      cmakeArgs: .. ${{ parameters.cMakeRunArgs }} -DCMAKE_TOOLCHAIN_FILE=../submodules/vcpkg/scripts/buildsystems/vcpkg.cmake -DENABLE_TESTING=1 -DCMAKE_BUILD_TYPE=${{ parameters.configuration }}
+      workingDirectory: 'build.${{ parameters.configuration }}'
+    displayName: Create build files
+  - task: CMake@1
+    inputs:
+      cmakeArgs: --build . --config ${{ parameters.configuration }}
+      workingDirectory: 'build.${{ parameters.configuration }}'
+    displayName: Build client
+  - bash: "./build.${{ parameters.configuration }}/bin/signalrclienttests"
+    condition: eq( variables['Agent.OS'], 'Darwin' )
+    displayName: Run tests
+  - bash: "./build.${{ parameters.configuration }}/bin/signalrclienttests"
```

Comment:
I'm not a fan of how this turns out in the Pipeline view:

![image](https://user-images.githubusercontent.com/7574/56064563-47ba2500-5d27-11e9-93c5-95b9802fc69b.png)

Can we make the specific test command conditional on `parameters.agentOs` like the `vmImage` is above?